resources/js/pages/auth/register.tsx

```
@@ -9,7 +9,7 @@ import { Spinner } from '@/components/ui/spinner';
 import { login } from '@/wayfinder/routes';
 import { store } from '@/wayfinder/routes/register';
 import { FieldDescription } from '@/components/ui/field';
-import { privacy, terms } from '@/wayfinder/routes/public/legal';
+import { privacy, terms } from '@/wayfinder/routes/home/legal';
```

ESLint

[error] `@/wayfinder/routes/home/legal` import should occur before import of `@/wayfinder/routes/register`

(import/order)